docs/introduction/09-third_party.rst

```diff
@@ -39,37 +39,32 @@ to you to check them because you need to avoid duplication:
     'aldryn_common',
     'aldryn_newsblog',
     'aldryn_people',
-    'aldryn_reversion',
```

Comment:
Revert all changes in 09-third_party.rst, we will not make any changes here now.

Reply:
aldryn_reversion file may not exist can you check it. For now, I left it. That is the one blocking it. Others are just duplicate conflicts.